docs/en/query_language/agg_functions/reference.md

```diff
@@ -895,47 +1207,197 @@ Type: `Float32`.
 !!! note "Note"
     If no values are passed to the function (when using `quantileTimingIf`), [NaN](../../data_types/float.md#data_type-float-nan-inf) is returned. The purpose of this is to differentiate these cases from cases that result in zero. See [ORDER BY clause](../select.md#select-order-by) for notes on sorting `NaN` values.
 
-The result is deterministic (it doesn't depend on the query processing order).
 
 **Example**
 
+Input table:
+
+```text
+┌─response_time─┬─weight─┐
+│            68 │      1 │
+│           104 │      2 │
+│           112 │      3 │
+│           126 │      2 │
+│           138 │      1 │
+│           162 │      1 │
+└───────────────┴────────┘
+```
+
+Query:
+
 ```sql
-SELECT quantileTiming(0.5)(number / 2) FROM numbers(10)
+SELECT quantileTimingWeighted(response_time, weight) FROM t
 ```
+
+Result:
+
 ```text
-┌─quantileTiming(0.5)(divide(number, 2))─┐
-│                                      2 │
-└────────────────────────────────────────┘
+┌─quantileTimingWeighted(response_time, weight)─┐
+│                                           112 │
+└───────────────────────────────────────────────┘
 ```
 
-## quantileTimingWeighted(level)(x, weight)
+**See Also**
 
-Differs from the `quantileTiming` function in that it has a second argument, "weights". Weight is a non-negative integer.
-The result is calculated as if the `x` value were passed `weight` number of times to the `quantileTiming` function.
+- [median](#median)
+- [quantiles](#quantiles)
 
-## quantileExact(level)(x)
 
-Computes the quantile of 'level' exactly. To do this, all the passed values ​​are combined into an array, which is then partially sorted. Therefore, the function consumes O(n) memory, where 'n' is the number of values that were passed. However, for a small number of values, the function is very effective.
+## quantileTDigest {#quantiletdigest}
 
-## quantileExactWeighted(level)(x, weight)
+Computes an approximate [quantile](https://en.wikipedia.org/wiki/Quantile) of a numeric data sequence using the [t-digest](https://github.com/tdunning/t-digest/blob/master/docs/t-digest-paper/histo.pdf) algorithm.
 
-Computes the quantile of 'level' exactly. In addition, each value is counted with its weight, as if it is present 'weight' times. The arguments of the function can be considered as histograms, where the value 'x' corresponds to a histogram "column" of the height 'weight', and the function itself can be considered as a summation of histograms.
+The maximum error is 1%.  Memory consumption is `log(n)`, where `n` is a number of values. The result depends on the order of running the query, and is nondeterministic.
```

Comment:
`The maximum error is 1%.` - error is not defined. We need to explain this better.
PS. In previous doc version it is the same.